docs/tutorial/10-resources-compose.mdx

```diff
@@ -7,19 +7,21 @@ In this tutorial, we're going to walk through how resources can own other resour
 ---
 
 <Callout type="info">
-
-Open the starter code for this tutorial in the Flow Playground:
-<a href="https://play.onflow.org/cda4d057-5898-4a7f-be34-fd47ec75085f" target="_blank">https://play.onflow.org/cda4d057-5898-4a7f-be34-fd47ec75085f</a><br/>
-The tutorial will be asking you do take various actions to interact with this code.
-
+  Open the starter code for this tutorial in the Flow Playground:
+  <a 
+    href="https://play.onflow.org/01f812d7-799a-42fd-b9cb-9ffe556e02ad"
+    target="_blank"
+  >
+    https://play.onflow.org/01f812d7-799a-42fd-b9cb-9ffe556e02ad
```

Comment:
I checked this link and it looks like the code still doesn't have the updated code that works with secure cadence in the 2.Tip Hat transaction. Can you make sure that that is updated and you have the correct link here?